common/src/main/java/com/microsoft/identity/common/adal/internal/tokensharing/TokenShareUtility.java

```diff
@@ -58,22 +58,20 @@
 import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.OAuth2.ID_TOKEN_OBJECT_ID;
 import static com.microsoft.identity.common.adal.internal.tokensharing.ITokenShareResultInternal.TokenShareExportFormatInternal.RAW;
 import static com.microsoft.identity.common.adal.internal.tokensharing.ITokenShareResultInternal.TokenShareExportFormatInternal.SSO_STATE_SERIALIZER_BLOB;
-import static com.microsoft.identity.common.java.exception.ClientException.TOKEN_CACHE_ITEM_NOT_FOUND;
 import static com.microsoft.identity.common.internal.migration.AdalMigrationAdapter.loadCloudDiscoveryMetadata;
 import static com.microsoft.identity.common.internal.migration.TokenCacheItemMigrationAdapter.renewToken;
 import static com.microsoft.identity.common.internal.migration.TokenCacheItemMigrationAdapter.sBackgroundExecutor;
+import static com.microsoft.identity.common.java.exception.ClientException.TOKEN_CACHE_ITEM_NOT_FOUND;
 
 public class TokenShareUtility implements ITokenShareInternal {
 
     private static final String TAG = TokenShareUtility.class.getSimpleName();
     private static final Map<String, String> sClaimRemapper = new HashMap<>();
     private static final String CONSUMERS_ENDPOINT = "https://login.microsoftonline.com/consumers";
-
-    /**
-     * To support caching lookups in ADAL, the following authority is used to signal
-     * that the tokens being yielded belong to the target user's home tenant.
-     */
-    private static final String sHomeTenantAuthority = "https://login.windows.net/common";
+    private static final String ENVIRONMENT_GLOBAL = "login.microsoftonline.com";
+    private static final String ENVIRONMENT_GALLATIN = "login.partner.microsoftonline.cn";
+    private static final String ENVIRONMENT_ITAR = "login.microsoftonline.us";
+    private static final String ENVIRONMENT_BLACKFOREST = "login.microsoftonline.de";
```

Comment:
Should we make this an enum, and then bake the authority in here?

Reply:
Applied in 48a2ac32baa9d8a894959676e23876143bd7bff6